test/adapter/json_api/relationship_test.rb

```diff
@@ -151,11 +151,8 @@ def test_relationship_with_everything
         private
 
         def test_relationship(expected, params = {})
-          options = params.fetch(:options, {})
-          links = params.fetch(:links, {})
-          meta = params[:meta]
           parent_serializer = AuthorSerializer.new(@author)
-          relationship = Relationship.new(parent_serializer, @serializer, options, links, meta)
+          relationship = Relationship.new(parent_serializer, @serializer, nil, params)
```

Comment:
As far as I've been able to understand, params here is the other options and `serializable_resource_options` are the options that come down through the serializable_hash options.